lib/handler/retry-handler.js

```diff
@@ -385,10 +398,11 @@ class RetryHandler {
     this.retryOpts.retry(
       err,
       {
+        controller,
         state: { counter: this.retryCount },
         opts: { retryOptions: this.retryOpts, ...this.opts }
       },
-      shouldRetry.bind(this)
+      shouldRetryCb
```

Comment:
because shouldRetryCb is now an arrow function, this points to the retry handler anyway.